src/AspComet/MessageConverter.cs

```diff
@@ -26,7 +26,14 @@ public static Message[] FromJson(HttpRequestBase request)
 
         private static string GetJsonFromMessageFieldIn(HttpRequestBase request)
         {
-            return request.Form["message"];
+            try
+            {
+                return request.Form["message"];

```

Comment:
Strange the documentation says that `NameValueCollection` wouldn't throw if the key is missing, it should return null

Reply:
i wasn't sure if the NCOver license allowed me to install a copy.. after looking at their outdated website I started to wonder what NCover might do to my V.S. 2017 installation.

So, I was hesitant to disturb your existing work. However, I'll be more than happy to do an in-place update to the project(s) and do a new pull request, if you prefer.

I also wasn't sure if people might prefer the older option in some scenarios, for whatever reason. Let me know what you prefer.

Reply:
regading request.Form .. maybe the .NET Core version of the HttpRequest is more finicky?
I wish I had written down the exception I was getting,